hs/src/Reach/Connector/ETH_Solidity.hs

```diff
@@ -662,6 +663,16 @@ solExpr sp = \case
           T_Object {} -> objPrefix
           _ -> impossible "objectref"
     return $ oe' <> "." <> p (pretty f) <> sp
+  DLE_ObjectSet _ obj_a fieldName val_a -> do
+    let objFields = M.fromList $ argObjstrTypes obj_a
+    result_t <- solType $ T_Object $ M.insert fieldName (argTypeOf val_a) objFields
```

Comment:
This should be the same as `obj_a`'s type

Reply:
Same situation as similar comment on ALGO.hs; not true if we're adding a new field or changing the field's type

Reply:
See prior comments about this